src/test/java/com/znsio/sample/e2e/screen/web/ajio/AjioHomeScreenWeb.java

```diff
@@ -32,7 +44,42 @@ public AjioSearchResultsScreen searchFor(String product) {
         searchElement.clear();
         searchElement.sendKeys(product);
         visually.checkWindow(SCREEN_NAME, "Search string entered");
-        driver.waitTillElementIsPresent(bySearchIconClassName);
+        driver.waitTillElementIsPresent(bySearchIconClassName).click();
         return AjioSearchResultsScreen.get();
     }
+
+    @Override
+    public AjioHomeScreen signInUser(String emailId, String password) {
+        LOGGER.info("Logging in as Valid user");
+        driver.waitForClickabilityOf(bySignInXpath).click();
+        driver.findElement(byEmailEntryClassName).sendKeys(emailId);
+        driver.findElement(byContinueClassName).click();
+        driver.waitForClickabilityOf(byLoginWithPasswordXpath).click();
+        driver.waitTillElementIsVisible(byPasswordEntryId).sendKeys(password);
+        driver.findElement(byStartShoppingXpath).click();
+        return this;
+    }
+
+    @Override
+    public boolean isUserSignedIn() {
+        LOGGER.info("Verifying user is logged in successfully");
+        driver.waitTillElementIsVisible(byMyAccountXpath);
+        visually.check(SCREEN_NAME, "Ajio Home page for LoggedIn User", Target.window().fully());
+        return driver.isElementPresent(byMyAccountXpath);
+    }
+
+    @Override
+    public boolean isUserLoggedOut() {
```

Comment:
keep the method names consistent, eg: for login the method name used is isUserSignedIn() and for logout its isUserLoggedOut()

Reply:
Updated to method name to be consistent